packages/buidler-core/test/fixture-projects/multiline-import-project/buidler.config.js

```diff
@@ -0,0 +1,3 @@
+require("../../../src/internal/lib/buidler-lib");
```

Comment:
```suggestion
```

Can we remove this? I guess you copy it from another project, but shouldn't be here, as it can break very easily.

Reply:
Removed 👍 Yes I just copied an existing fixture.